src/App.js

```diff
@@ -25,12 +26,14 @@ function App () {
       <PublicRoute path='/resetpassword' component={ResetPasswordForm} />
       <PublicRoute path='/setnewpassword' component={SetNewPasswordForm} />
       <PublicRoute path='/store/:id' component={Store} />
+      <PublicRoute path='/payment' component={Stripe} />
```

Comment:
@kip-guile There are 2 routes, Remove one

Reply:
Apart from this minor tweak it looks fine to me.